src/frontend/src/pages/index.js

```diff
@@ -88,7 +88,7 @@ export default function IndexPage() {
     setNumPages(numPages + 1);
   }
 
-  async function getPostsCount() {
+  const getPostsCount = useCallback(async () => {
```

Comment:
I'm unclear if you can use an `async` function here.  What I read online and in the docs doesn't help much.

Reply:
https://stackoverflow.com/questions/60835660/proper-use-of-usecallback . Looks like this guy does it in the React hook